packages/client/src/components/signin.vue

```diff
@@ -45,190 +48,185 @@
 </form>
 </template>
 
-<script lang="ts">
-import { defineAsyncComponent, defineComponent } from 'vue';
+<script lang="ts" setup>
+import { defineAsyncComponent, defineEmits, defineProps } from 'vue';
```

Comment:
> `defineProps` and `defineEmits` are compiler macros only usable inside `<script setup>`. They do not need to be imported, and are compiled away when `<script setup>` is processed.

https://vuejs.org/api/sfc-script-setup.html#defineprops-defineemits
```suggestion
import { defineAsyncComponent } from 'vue';
```